scripts/tests/spawn_network.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+if [ ! -d "./scripts/tests" ]; then
+    echo "Please execute this script from the root of the Zeitgeist project folder"
+    exit 1
+fi;
+
+cargo build --features parachain
+
+# Define destination path
+DEST_PATH="./tmp/zombienet"
```

Comment:
There is no need for spawning a `tmp` directory in the project root (which then has to be cleaned up manually), as Unix systems provide a temporary folder at the system root `/tmp`. Any user can write into that and contents get cleaned up after shutdown. More info at https://linuxhandbook.com/tmp-directory/

Reply:
But do we want that our zombienet and polkadot executable is cleaned up after the next shutdown? My expected usage would be to not download the zombienet executable after every startup. I mean we don't push the files inside the tmp folder to github, since the folder is added to the gitignore.

Reply:
I agree this would not be optimal. Maybe rename `tmp` into `bin` then, as `tmp` indicates temporary files that will be removed after usage or a predefined time period.

Additionally, I recommend to also download Polkadot into the bin folder (if you want to can toggle this via a variable as well), such that everything the script needs to spawn a parachain zombienet is available.